src/bindgen/ir/ty.rs

```diff
@@ -592,50 +516,14 @@ impl Type {
         }
     }
 
-    fn nonzero_to_primitive(&self) -> Option<Self> {
```

Comment:
This function is no longer needed, because the parser handles these types directly now.